src/App/Header/UserMenu/UserMenu.tsx

```diff
@@ -0,0 +1,24 @@
+import { Authenticator, useAuthenticator } from "@aws-amplify/ui-react";
+import { Popover } from "@headlessui/react";
+import Icon from "components/Icon";
+import Menu from "./Menu";
+
+export default function UserMenu() {
+  const { user, signOut } = useAuthenticator((context) => [context.route]);
```

Comment:
(Possible) Bug - if a user uses the top-right "LOGIN" button for registration, once they register they *seem* to be signed in even w/o confirming their email, but when they navigate to the Admin Panel, they are asked to sign in again and when they do then they are asked to confirm their email.

Expectation: the user should be asked to confirm their email immediately on registration even when using the top right LOGIN button. This requirement needs to be indicated to the user somehow.

Repro video below (ignore Loom notifications):
https://www.loom.com/share/6c8100dfcb664bf58b07da298011da36?sid=e08239e7-e0c6-4989-81c1-80e75e797457

Reply:
this is also fixed by 
https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2427/commits/736ebd64e871d995151f5e4c9776a862d68edaca